src/modules/packages.c

```diff
@@ -77,6 +105,8 @@ void ffPrintPackages(FFinstance* instance)
         if(name > 0) \
         { \
             printf("%u ("#name")", name); \
+            if(name == pacman)  \
+                getActiveBranch(); \
```

Comment:
This seems unsafe, what happens if a package manager has the same amount of packages as pacman? Just don't use the macro for pacman and write it by hand.

Reply:
Oops. I was seeing  'pacman' as an enum I could use to identify it.

Reply:
How do I get the output in front of the comma without modifying the macro?

Reply:
Just replace `FF_PRINT_PACKAGE(pacman)` with
```c
if(pacman > 0)
{
    printf("%u (pacman)", pacman);
    if(manjaroBranch.length > 0)
        printf("[%s]", manjaroBranch.chars);
    if((all = all - pacman) > 0)
        printf(", ");
};
```